assembl/models/post.py

```diff
@@ -560,6 +560,10 @@ def restrict_to_owners(cls, query, user_id):
         "filter query according to object owners"
         return query.filter(cls.creator_id == user_id)
 
+    def is_bright_mirror_fiction(self):
+        parent_idea = [link.idea for link in self.indirect_idea_content_links_without_cache() if link.__class__.__name__ == 'IdeaRelatedPostLink']
+        return len(parent_idea) != 0 and parent_idea[0].message_view_override == 'brightMirror'
```

Comment:
Check against the `Phases` enum type. Keeping things DRY.

Reply:
No, don't check against phase, only on message_view_override. We can have in the future a thread phase with a single idea in brightMirror mode. If you mean using the Phases enum instead of 'brightMirror', this is wrong, you should create another enum with only multiColumns and brightMirror as possible values.

Reply:
Yeah sure, why not.

```
brightMirrorables = {
   Phases.brightMirror,
   Phases.multiColumns
}
```

You can put it in `post.py` and check that the `message_view_override` is in the `brightMirrorables`.

Reply:
So create in `models/idea.py` a new enum
```
class MessageView(Enum):
    multiColumns = 'multiColumns'
    brightMirror = 'brightMirror'
```